docsearch/algolia_configs/latest_commit.json

```diff
@@ -1,5 +1,6 @@
 {
     "en-tidb-dev": "9680a4b7e1e708f90341dcbb0795247e64ba441a",
+    "en-tidb-v8.2": "to-be-updated",
```

Comment:
to-be-updated according to the latest commit in https://github.com/pingcap/docs/commits/release-8.2/